src/test/kotlin/io/github/mojira/arisa/modules/RemoveNonStaffMeqsModuleTest.kt

```diff
@@ -22,7 +23,16 @@ class RemoveNonStaffMeqsModuleTest : StringSpec({
 
     "should return OperationNotNeededModuleResponse when there is no comments with an MEQS tag" {
         val module = RemoveNonStaffMeqsModule("")
-        val comment = Comment("I like QC.", null, null) { Unit.right() }
+        val comment = Comment(
+            "I like QC.",
+            "",
+            { emptyList() },
+            Instant.now(),
+            Instant.now(),
+            null,
+            null,
+            { Unit.right() },
+            { Unit.right() })
```

Comment:
(Trivial) (Subjective) (WAI) The last parentheses drives me crazy lol

Reply:
Don't we have a linter for this sort of stuff? Is it not working properly, or does it not have a rule for this sort of stuff?

Reply:
No rule, apparently